content/blog/2018/08/2018-08-14-code-coverage-api-plugin-1.0-release.adoc

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+title: "Code Coverage API plugin: 1.0 Release"
+tags:
+- plugins
+- gsoc
+author: shenyu_zheng
+---
+
+I am happy to announce availability of Code Coverage API and LLVM-cov plugins. These plugins have been recently released as 1.0, and they are now available in the Jenkins Update Center. In this blogpost I will introduce the features and project structure of Code Coverage API plugin.
+
+=== About me
+
+My name is Shenyu Zheng, and I am an undergraduate student in Computer Science and Technology at Henan University from China.
+
+=== Overview
+
+Code Coverage API plugin is one of https://jenkins.io/projects/gsoc/[GSoC 2018 Jenkins projects].
+
+There are a lot of plugins which currently implement code coverage, however, they all use similar config, charts, and content. So it will be much better if we can have an API plugin which does the most repeated work for those plugins and offers a unified APIs which can be consumed by other plugins and external tools.
+
+My mentors are https://github.com/christ66[Steven Christou], https://github.com/Supun94[Supun Wanniarachchi], https://github.com/jeffpearce[Jeff Pearce] and https://github.com/oleg-nenashev[Oleg Nenashev].
+
+=== Supported Tools
+
+* Embedded tools
+** JaCoCo
+* Other plugins as an Extension of Code Coverage API plugin
+** Cobertura (https://github.com/jenkinsci/cobertura-plugin[Cobertura Plugin])
+** llvm-cov (https://github.com/jenkinsci/llvm-cov-plugin[llvm-cov Plugin])
+
+=== Features
+
+* Modernized coverage chart
+* Coverage trend
+* Source code navigation
+* Parallel pipeline support
+* Reports combining
+* Rest API
+* Failed conditions and flexible threshold setting
+* Other small features
+
+==== Modernized coverage chart
+
+In the summary chart we can see the coverage summary of current coverage metric.
+image:/images/post-images/2018-06-13-code-coverage-api-plugin/summary-chart.gif[title="Summary Chart", role="center"]
+
+==== Child Summary Chart
+
+In the Child summary chart, we can see the coverage summary of each child, also, we can use the range handler to filter item we want to see to reduce the chart size. If we want see coverage details of child, we can click the child name to see more information.
+image:/images/post-images/2018-06-13-code-coverage-api-plugin/child-summary-chart.gif[title="Child Summary Chart", role="center"]
+
+=== Coverage Trend
```

Comment:
You should be consistent in capitalization in these sections. Here `Trend` is capitalized, but in most of the sections, on the first word is capitaliazed